atlasdb-timelock-server/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosResource.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.timelock.paxos;
+
+import java.nio.file.Paths;
+import java.util.Map;
+
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Maps;
+import com.palantir.paxos.PaxosAcceptor;
+import com.palantir.paxos.PaxosAcceptorImpl;
+import com.palantir.paxos.PaxosLearner;
+import com.palantir.paxos.PaxosLearnerImpl;
+
+@Path("/{client: [a-zA-Z0-9_-]+}")
+public final class PaxosResource {
+    public static final String DEFAULT_LOG_DIRECTORY = "var/data/";
```

Comment:
(would that be `"./var/data/"`? Sorry, not sure I understand this - I thought this is already relative to the directory in which TimeLock runs)